x-pack/plugins/maps/public/components/global_filter_checkbox.tsx

```diff
@@ -21,13 +22,21 @@ export function GlobalFilterCheckbox({ applyGlobalQuery, label, setApplyGlobalQu
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalQuery}
-        onChange={onApplyGlobalQueryChange}
-        data-test-subj="mapLayerPanelApplyGlobalQueryCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalFilterHelp', {
+          defaultMessage:
+            'When enabled, the query from the global search bar is applied to this layer"',
```

Comment:
GlobalFilterCheckbox is used 2 places, layer editor and join editor. As such, the phrase "this layer"should not be hard coded in the message.

How about "When enabled, results narrowed global search"